src/librustdoc/html/static/js/storage.js

```diff
@@ -236,6 +241,12 @@ if (getSettingValue("use-system-theme") !== "false" && window.matchMedia) {
     switchToSavedTheme();
 }
 
+if (getSettingValue("source-sidebar-show") !== "false") {
```

Comment:
If it isn't set, then it'll be different than `false`. The correct condition should be this I think:

```suggestion
if (getSettingValue("source-sidebar-show") === "true") {
```